cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -653,6 +653,13 @@ permissions:
     - operations
     - viewer
 
+  labels_list:
```

Comment:
plz do test it with a user different than admin, because the storage_manager also adds its own permissions checking, in addition to `@authorize`, based on the model name, but I'm not sure if it still does it, if the permission is undeclared at all.

I'm worried whether it would also look for a `deployment_labels_key`, which is similar to an issue we've had when we first introduced... what was that model, I think Operations?

Reply:
Thanks, checked it for a user with a tenant role of `viewer` and a system role of `defualt` and it worked.